components/pipeline-service/production/kflux-rhel-p01/deploy.yaml

```diff
@@ -2078,7 +2080,7 @@ metadata:
 spec:
   dataFrom:
   - extract:
-      key: production/platform/ansible/generated/kflux-rhel-p01/chains-signing-secret
+      key: production/pipeline-service/kflux-rhel-p01/chains-signing-secret
```

Comment:
this should not be changed, the change was done directly in deploy.yaml file, which was wrong and fixed in https://github.com/redhat-appstudio/infra-deployments/pull/6572